src/System.Security.AccessControl/src/System/Security/AccessControl/SecurityDescriptor.cs

```diff
@@ -669,10 +669,8 @@ private static byte[] BinaryFormFromSddlForm(string sddlForm)
 nameof(sddlForm));
                     }
                     else if (error != Interop.Errors.ERROR_SUCCESS)
-                    {
-                        Debug.Assert(false, string.Format(CultureInfo.InvariantCulture, "Unexpected error out of Win32.ConvertStringSdToSd: {0}", error));
-                        // TODO : This should be a Win32Exception once that type is available
-                        throw new Exception();
+                    {                        
+                        throw new Win32Exception(error,SR.Format(SR.AccessControl_UnexpectedError,error));
```

Comment:
Style: There should be spaces after the commas.

Reply:
done